Documentation/source/guide/tutorials-imagesequence.rst

```diff
@@ -0,0 +1,209 @@
+
+
+.. note:: "-pix_fmt rgb24 or rgba" is a must to include convert the color space. YUV/YCRB is not ideal for many en/decoders for TIFF.
+
+
+For more information on support pix format and compression for TIFF in ffmpeg in the terminal type: ``ffmpeg -v error -h encoder=tiff``
+
+FFmpeg GUIs
+~~~~~~~~~~~
+Here are some graphical user interfaces (GUI) which can be used instead of the terminal
+
+- `ffmulticonv <https://sourceforge.net/projects/ffmulticonv/>`__ (Linux)
+- `WinFF <https://github.com/WinFF/winff>`__ (Windows and Mac)
+
+
+More information of FFmpeg's command line options https://ffmpeg.org/ffmpeg-formats.html
+
+
+Kdenlive/Shotcut
+~~~~~~~~~~~~~~~~
+https://kdenlive.org/
+https://shotcut.org/
+
+With the video(s) on the timeline go to Project > Render.
+In the render settings choose Images sequence and select the desired image format.
+
+The sequence will be output with the specified file name and format and using five digits for its numbered sequence (e.g. output_00001.png).
+
+Full instructions on how to use Kdenlive can be found here https://userbase.kde.org/Kdenlive/Manual/Project_Menu/Render
+
+Blender
+~~~~~~~
+https://www.blender.org/
+
+- Import the movie file in Blender Video Sequencer.
+- Go to render properties
+- In Color management change view transform standard
+- GO to output properties
+- Select File format PNG/TIFF
+- Select RGB/RGBA, 8/16 Color depth, and preferred compression NONE/Any.
+
+Full instructions on how to use the Blender VSE can be found here https://docs.blender.org/manual/en/latest/video_editing/index.html
+
+
+Adobe Media Encoder
+~~~~~~~~~~~~~~~~~~~
+- Open media encoder
+- Add source video to the queue
+- Set the output format to OpenEXR.
+- Set compression to "Zip"
+- If the source has an alpha channel be sure to scroll down to the bottom of the Video section of the Export Settings and check "Include Alpha Channel"
+- Close the Export Settings by clicking Ok and press the Start Queue button.
+
+How To Convert Image Sequences To Video Files
+=============================================
+
+FFmpeg
+~~~~~~
+Converting your images to video follows a similar process to doing the reverse.
+
+Open a terminal and navigate to the location containing the images.
+In the terminal type:
+
+``ffmpeg -i input_%05d.png output.mp4``
+
+Change ``input_`` to match the name of the files. The number of characters in the sequence (%05d) should match the amount in your input files. For example, if the files have four characters in their sequence (e.g. input_0001.png) then it should use %04d.
+
+For this to work correctly all of the files need to be sequentially numbered and the sequence should start from either 0 or 1.
+
+It is can also specify the framerate and the codec, here is an example for framerate 30fps:
+
+``ffmpeg -framerate 30 -i input%04d.png -c:v libx264 -r 30 -pix_fmt yuv420p out.mp4``
+
+Using an NLE
+============
+
+Kdenlive
+~~~~~~~~
+
+Shotcut
+~~~~~~~
+
+Da Vinci Resolve
+~~~~~~~~~~~~~~~~
```

Comment:
Steps:
- Open Davinci Resolve.
- Create a new project (temporary).
- Switch to Media page using one of these methods:
  + At the bottom, click the "Media" page.
  + Shift + 2.
- Import your video(s) using one of these methods:
  + File > Import File > Import Media.
  + Right click the Media Pool > Import Media.
  + Ctrl + I.
- Transcode your video(s) to a image sequence(s) using Media Management:
  + File > Media Management.
  + On the top of the pop up menu, click Entire Project.
  + Click Transcode.
  + Choose your destination folder under Media Destination.
  + Choose "Transcode all media".
  + Change video format to EXR or TIFF
  + Set detail settings as you desire.
  + Click "Start" to start transcoding.